lambda/action_network.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import os
+from typing import Dict
+
+import requests
+from dateutil import parser
+
+log = logging.getLogger(__name__)
+
+
+def get_object_or_empty_dict(key: str, event: Dict) -> Dict:
+    location = event.get(key)
+    if not location:
+        location = {}
+    return location
+
+
+def make_location(event: Dict) -> str:
+    location = get_object_or_empty_dict('location', event)
+    if location.get('country', 'US') != 'US':
+        return '{city}, {country}'.format(
+            city=location.get('locality'), country=location.get('country'))
+    return location.get('postal_code', '')
+
+
+def get_event_name(event: Dict) -> str:
+    name = event.get('name', None)
+    if not name:
+        name = event.get('title', '')
+    return name
+
+
+def get_organizer(event: Dict) -> Dict:
+    return get_object_or_empty_dict('osdi:organizer',
+                                    get_object_or_empty_dict(
+                                        '_embedded', event))
+
+
+def get_contact_name(event: Dict) -> Dict:
+    organizer = get_organizer(event)
+    given_name = organizer.get('given_name')
+    family_name = organizer.get('family_name')
+    return ' '.join([given_name, family_name])
```

Comment:
`return '%s %s' % (organizer.get('given_name', ''), organizer.get('family_name', ''))`

need a default '' in case one of these is None; don't want 'None Smith'

Reply:
I would agree, an put the '' default in gets, but otherwise I disagree.  Furthermore, the way you wrote this would result in <space>Smith.  Using the join would give us just Smith.

Reply:
I prefer the reality to the conciseness.